lib/cli/utils.ts

```diff
@@ -28,6 +36,19 @@ export function getPackageName(manifest): string {
     }
     return (manifest.namespace || manifest.name).toLowerCase();
 }
+/**
+ * @description Retrieves the LabShare package's lsc settings
+ * @param {Object} manifest - A package.json parsed into a JS object
+ */
+export function getPackageLSCSettings(manifest): ILSCSettings {
```

Comment:
input type missing.
rename to follow camelCase https://docs.microsoft.com/en-us/previous-versions/dotnet/netframework-1.1/141e06ef(v=vs.71)?redirectedfrom=MSDN